.claude/epics/timestamp-remediation/532.md

```
@@ -0,0 +1,95 @@
+        ]
+        result = resolver.resolve(prefs)
+        assert result is not None
+
+    def test_operation_history_timestamp_format(self):
+        """Operation history timestamps use Z suffix, not +00:00Z."""
+        ts = datetime.now(UTC).isoformat().replace("+00:00", "Z")
+        assert ts.endswith("Z")
+        assert "+00:00Z" not in ts
+```
+
+## Dependencies
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)